06_modules_lab/02_bonus.py

```diff
@@ -1,15 +1,12 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-Bonus: Print error messages for invalid inputs.
-
-To print error messages we'll have to use a concept not yet learned in the
-course, and which will only be presented later: Exceptions.
-We'll tap into python's error handling and change its default
-error message to something more meaningful.
-
-Read ahead the exception chapter here:
-https://docs.python.org/2.7/tutorial/errors.html
-
-And then continue to writing the code
-"""
-
+""" get ywo nuber and print their sum + check"""
+import os,sys
+if len(sys.argv) != 3 :
+    print "we need exactlly 2 numbers"
+    quit()

```

Comment:
quit is not the recommended way to leave a python script in the middle, because it doesn't report error to the calling program or operating system.

The better way is sys.exit, which also received the error message as argument:

```
sys.exit('we need exactly 2 numbers')
```